src/runtime/entitlements-manager-test.js

```diff
@@ -79,6 +78,7 @@ describes.realWin('EntitlementsManager', {}, env => {
     xhrMock.verify();
     jwtHelperMock.verify();
     analyticsMock.verify();
+    eventManagerMock.verify();
```

Comment:
Nothing seems to have any expectations of event manager mock any more.  Should it just go away?

Reply:
Yes I removed it. I still need to instantiate a ClientEventManager here because AnalyticsService uses it though.